Solutions/Web Session Essentials/Hunting Queries/RequestFromBotsAndCrawlers.yaml

```diff
@@ -0,0 +1,41 @@
+id: b653c2e0-ae0f-490b-9b2e-836858d28976
+name: Request from bots and crawlers (ASIM Web Session)
+description: |
+  'This query searches for requests from bots and crawlers.'
+tags:
+  - Schema: WebSession
+    SchemaVersion: 0.2.6
+requiredDataConnectors: []
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  let lookback =- 1d;
+  // You can add or remove items from this list as per your requirement.
+  let BotandCrawlersList = dynamic (['bot','crawler','baiduspider','80legs','ia_archiver','voyager','wget','yahoo','slurp','mediapartners-google','facebookexternalhit','facebookcatalog','okhttp','cURL','Bytespider']);
+  _Im_WebSession (starttime=ago(lookback),httpuseragent_has_any= BotandCrawlersList)
+  | summarize EventCount = count(), EventStartTime=min(TimeGenerated), EventEndTime=max(TimeGenerated), UserAgentList = make_set(HttpUserAgent,100), URLs = make_set(Url,100) by SrcIpAddr, SrcUsername, SrcHostname
```

Comment:
We would require to remove the empty User Agent rows.

Reply:
there seems to be some issue with filter. not using filter currently to avoid rule malfunction. Will resolve filter separately